benchmarks/utilities/log_parser.hpp

```diff
@@ -21,11 +21,11 @@
 #include <limits>
 #include <memory>
 #include <rmm/detail/error.hpp>
+#include <rmm/mr/device/device_memory_resource.hpp>
```

Comment:
Let's group the headers with a blank line between each group (like we do most other places in RMM and cuDF).  RMM headers first, then rapidcsv.h, then C++ std headers.

Reply:
Sure, done. I had gone with the minimal change at first, but then clang-format ended up reordering things, so that's a good reminder to group things.